buku

```diff
@@ -102,14 +102,26 @@ class BMHTMLParser(HTMLParser.HTMLParser):
 
 
 
+def getDataPath():
+    data_home = os.environ.get('XDG_DATA_HOME')
+    if data_home is None:
+        if os.environ.get('HOME') is None:
+            data_home = '.'
+        else:
+            data_home = os.path.join(os.environ.get('HOME'), '.local', 'share')
+
+    return os.path.join(data_home, 'buku')
+
+
+
 def initdb():
```

Comment:
Hi,

Please bump the version to 1.9 and add a minor routine to move the existing bookmarks.db file if version is greater than 1.8. Otherwise we will break `buku` for people who are already using it.